test/sqllogictest/network_policy.slt

```diff
@@ -0,0 +1,82 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Test for `NETWORK POLICIES`.
+
+mode standard
+
+# Start from a pristine state
+reset-server
+
+simple conn=mz_system,user=mz_system
+
+query TT rowsort
+select * from (SHOW NETWORK POLICIES)
+----
+db error: ERROR: Unexpected keyword QUERY at the beginning of a statement
+
+
+simple conn=mz_system,user=mz_system
+ALTER SYSTEM SET enable_network_policies=on;
+----
+COMPLETE 0
+
+
+statement ok
+CREATE NETWORK POLICY np (RULES (  r1 (address='0.0.0.0/32', action='allow', direction='ingress'), r2 (address='0.0.0.1/32', action='allow', direction='ingress') ));
+
+query TT rowsort
+SHOW NETWORK POLICIES
+----
+np
+(empty)
```

Comment:
should `SHOW NETWORK POLICIES` include more info? maybe the rules that are in the policy?

Reply:
I updated this to include rules, for better or worse, I went with a similar model to how we display replicas on clusters.  The downside to this is that one could create a policy with 25 rules, leading to a rather long value.

Reply:
That looks good to me!